model_utilities/train/finetuner.py

```diff
@@ -43,52 +44,83 @@ def __plot_losses(self, training_losses: list[float], validation_losses: list[fl
         plt.legend()
         plt.savefig("./debug/graphs/losses.png")
 
-    def __train(self, model: AutoModelForSeq2SeqLM, tokenizer: NllbTokenizer, dataset: ds.Dataset, optimizer: Adafactor,
+    def __tokenize_dataset(self, dataset: ds.Dataset, tokenizer: NllbTokenizerFast, max_length: int) -> Dataset:
+        # Not meant to be called outside of this function
```

Comment:
This is an unnecessary comment. `tokenize` inside `__tokenize_dataset` clearly indicates that.